TestCntlrApp/src/tfwApp/fw_api_int.x

```diff
@@ -1467,6 +1467,7 @@ typedef struct _PartialReset
 {
    U32 numOfConn;
    UeS1apIdPair *ueS1apIdPairList;
+   U32 *ueIdLst;
```

Comment:
Hi @sudheesh001  Is this parameter used? because fw_api_init.c; I could see this parameter is not being used.

1593    else if(msgReq->t.resetReq.rstType == NB_PARTIAL_RESET)
1594    {
1595       msgReq->t.resetReq.u.partialRst.numOfConn = data->r.partialRst.numOfConn;
1596       FW_ALLOC_MEM(
1597           fwCb, &msgReq->t.resetReq.u.partialRst.ueS1apIdPairList,
1598           sizeof(NbUeS1apIdPair) * msgReq->t.resetReq.u.partialRst.numOfConn);
1599
1600       cmMemcpy(
1601           msgReq->t.resetReq.u.partialRst.ueS1apIdPairList,
1602           data->r.partialRst.ueS1apIdPairList,
1603           sizeof(NbUeS1apIdPair) * msgReq->t.resetReq.u.partialRst.numOfConn);
1604    }
1605    else
1606    {
1607       FW_LOG_ERROR(fwCb, "Invalid reset Type");
1608       FW_LOG_EXITFN(fwCb, RFAILED);
TestCntlrApp/src/tfwApp/fw_api_int.c 

Once Reset message is received at test framwork application, later posts the message to eNB application; Even while it get posted. we don't see this parameter. Below is the structure that gets populated while posted to eNB application
231 typedef struct _mnPartialReset
232 {
233    U32 numOfConn;
234    NbUeS1apIdPair *ueS1apIdPairList;
235 }NbPartialReset;
236

Reply:
Hi @rsarwad Thank you for the review. We do see this in the `TestCntlrStub` which fails to build at these lines where `ueIdLst` is used.

```
((ResetReq*)msgptr)->r.partialRst.numOfConn = numOfUes;
((ResetReq*)msgptr)->r.partialRst.ueIdLst = malloc(numOfUes);
memcpy(((ResetReq*)msgptr)->r.partialRst.ueIdLst, ueLst, numOfUes);
```
Given what you mentioned, it looks like the Stub might be out of date and should rather use `UeS1apIdPair` by doing something like:
```
((ResetReq*)msgptr)->r.partialRst. ueS1apIdPairList = malloc(sizeof(UeS1apIdPair) * numOfUes);
memcpy(((ResetReq*)msgptr)->r.partialRst. ueS1apIdPairList, modifiedUeLst, numOfUes);
```

Reply:
Yes right